main.js

```diff
@@ -1,28 +1,79 @@
-function generateBoard() {
-  var $board = document.createElement('div')
+function Tile(row, column, isHidden) {
```

Comment:
Stay away from constructor functions unless you plan to add things to its prototype. Just stick with plain functions that return plain objects `{}`.